config/eslint.config.mjs

```diff
@@ -50,6 +50,19 @@ export default [
           "message": "The use of Buffer is not allowed."
         }
       ],
+        "no-restricted-properties": [
+          "warn",
+          {
+            "object": "assert",
+            "property": "ok",
+            "message": "Usage of assert.ok is discouraged because it relies on truthiness."
```

Comment:
Should we scope this to the `./packages/*/test`? It is now monorepo-wide (which is fine) but it also includes the src folder. I think Node itself also has an assert library (which I doubt we use). But this should be scoped for tests-only right?

Reply:
Will move it down to one of the confined rule sets that includes tests.